nodejs/script.sh

```diff
@@ -0,0 +1,47 @@
+
+
+transmute scitt key generate \
+--alg -35 \
+--output private-key.cbor
+
+transmute scitt key export \
+--input  private-key.cbor \
+--output public-key.cbor
+
+# transmute scitt key diagnose \
+# --input  public-key.cbor \
+# --output artifact.manifest.spdx.json.issuer.public-key.cbor.md
+
+transmute scitt statement issue \
+--iss urn:example:123 \
+--sub urn:example:456 \
+--issuer-key private-key.cbor \
+--statement  artifact.manifest.spdx.json \
+--signed-statement artifact.manifest.spdx.json.cbor
+
+# transmute scitt statement diagnose \
+# --input  artifact.manifest.spdx.json.cbor \
+# --output artifact.manifest.spdx.json.cbor.without-transparency.md
+
+transmute scitt statement verify \
+--issuer-key public-key.cbor \
+--statement  artifact.manifest.spdx.json \
+--signed-statement artifact.manifest.spdx.json.cbor \
+
+transmute scitt ledger receipt issue \
+--iss urn:example:789 \
+--sub urn:example:abc \
+--issuer-key private-key.cbor \
+--signed-statement  artifact.manifest.spdx.json.cbor \
+--transparent-statement artifact.manifest.spdx.json.cbor \
+--ledger ledger.json
```

Comment:
Minor, non-blocking: not to nit-pick, but do we want to consider making this `--log`? I guess `--transparency-service-log` or `--append-only-log` seem equally ridiculous but given the current architecture draft, I guess I can see why we stick to ledger. It's definitely easy to write code interfaces against. 😢